.github/SECURITY.md

```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+This security policy applies to all public repositories under the [OpenBoxes](https://github.com/openboxes) organization on 
+GitHub.
+
+## Supported Versions
+We provide security updates for the [latest official release](https://github.com/openboxes/openboxes/releases/latest) of OpenBoxes.
+
+## Reporting a Vulnerability
+
+If you discover a vulnerability in any public project under the [OpenBoxes](https://github.com/openboxes) organization, 
+please report it responsibly to [support@openboxes.com](mailto:support@openboxes.com).
```

Comment:
We can also allow people to report a vulnerability by going to https://github.com/openboxes/openboxes/security